manifests/postgres/sql/runservices.sql

```diff
@@ -0,0 +1,116 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.8 (Debian 12.8-1.pgdg110+1)
+-- Dumped by pg_dump version 12.8 (Debian 12.8-1.pgdg110+1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: run_database; Type: DATABASE; Owner: admin
+--
+
+CREATE DATABASE run_database;
+\connect run_database; 
+
+--
+-- Name: run_schema; Type: SCHEMA; Schema: -; Owner: admin
+--
+
+CREATE SCHEMA run_schema;
+
+
+ALTER SCHEMA run_schema OWNER TO admin;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: run_number; Type: TABLE; Schema: run_schema; Owner: admin
+--
+
+CREATE TABLE run_schema.run_number (
```

Comment:
I think this table still misses the first entry for run number, at least I cannot see it.

Reply:
I mean the INSERT statement, with the first run number we want to use.